polaris-react/src/components/Tile/tests/Tile.test.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import {mountWithApp} from 'tests/utilities';
+
+import {Tile} from '../Tile';
+
+const text = 'This is a tile';
+const children = <p>{text}</p>;
+
+describe('<Tile />', () => {
+  it('renders children', () => {
+    const tile = mountWithApp(
+      <Tile spacing="1" columns="1">
+        {children}
+      </Tile>,
+    );
+
+    expect(tile).toContainReactComponent('p', {children: text});
```

Comment:
```suggestion
    expect(tile).toContainReactComponent(children);
```
I'm not sure if this will work but it would mean we could merge the two constants above

Reply:
I gave this a shot and the `.toContainReactComponent` function starts to complain 😕. According to [this](https://github.com/Shopify/quilt/blob/main/packages/react-testing/README.md#-tocontainreactcomponenttype-string--reactcomponenttype-props-object) the first argument passed has to be a string

Reply:
You can do:
remove `const text`.
`const Children = () => <p>This is a tile</p>`
```suggestion
    expect(tile).toContainReactComponent(Children);
```

You'll also have to update the way children is rendered on in your tests from `{children}` to `<Children />`. `.toContainReactComponent()` expects the first arg to be a string or a component.

Reply:
Thats works!